weblate/templates/snippets/sort-field.html

```diff
@@ -21,10 +21,10 @@
   </div>
   <div class="btn-group" role="group">
     <button type="button" class="btn btn-default search-field query-sort-toggle">
-      <span class="search-icon asc active">
+      <span title="Ascending" class="search-icon asc active">
```

Comment:
These should be translatable:

```suggestion
      <span title="{% trans "Ascending" %}" class="search-icon asc active">
```